src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -357,19 +359,36 @@ func (cfg Config) doFetchCloudConfig(proxyAddr string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Unable to initialize HTTP client: %s", err)
 	}
+	country := strings.ToLower(globals.GetCountry())
```

Comment:
TBH I didn't think that getting it picked up on the next pass was much of a problem.  I'll make this wait for a non-"" country.